lib/vmdb/settings/validator.rb

```diff
@@ -184,6 +195,16 @@ def smtp(data)
           errors << [:from, "\"#{data.from}\", invalid. Should be a valid email address"]
         end
 
+        if send_notifications && @roles.split(",").include?("notifier") && data.domain == "mydomain.com"
+          message = "SMTP domain is invalid. Notifier role is enabled but smtp domain has not be changed from the defaults.  Disable 'notifier' server role if email notification is not needed."
+          notify_superadmin(message, nil, resource)
+        end
+
+        if send_notifications && !@roles.split(",").include?("notifier") && data.domain != "mydomain.com"
```

Comment:
We should probably change the default, and check for blank and any default values and treat them as "not configured"

Reply:
Also, I picked domain, but are there other defaults we can check to see if they didn't configure smtp?